tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -101,34 +101,34 @@ describe('Header', () => {
     expect(welcomeTab.hasClass('usa-current')).toEqual(true)
   })
 
-  it("should add usa-current class to Reports tab when on '/reports'", () => {
+  it("should add usa-current class to Reports tab when on '/data-files'", () => {
     const store = mockStore({
       ...initialState,
-      router: { location: { pathname: '/reports' } },
+      router: { location: { pathname: '/data-files' } },
     })
     const wrapper = mount(
       <Provider store={store}>
         <Header />
       </Provider>
     )
 
-    const reportsTab = wrapper.find('#reports')
+    const reportsTab = wrapper.find('#data-files')
 
     expect(reportsTab.hasClass('usa-current')).toEqual(true)
   })
 
-  it("should add usa-current class to Reports tab when on '/reports/*'", () => {
+  it("should add usa-current class to Reports tab when on '/data-files/*'", () => {
     const store = mockStore({
       ...initialState,
-      router: { location: { pathname: '/reports/upload' } },
+      router: { location: { pathname: '/data-files/upload' } },
     })
     const wrapper = mount(
       <Provider store={store}>
         <Header />
       </Provider>
     )
 
-    const reportsTab = wrapper.find('#reports')
+    const reportsTab = wrapper.find('#data-files')
```

Comment:
A find and replace for reports => dataFiles might take care of these in this file.

Reply:
Nice catch, thanks!